### PR TITLE
Variable Initialization for braket pipeline benchmarks

### PR DESCRIPTION
**Pull request by albi3ro**
This PR changes the parameter initialization in the `pl-braket-pipeline.py` benchmarks.  

For the casual one, the parameters are randomly generated in the [updated numpy style](https://numpy.org/doc/stable/reference/random/index.html?highlight=random#module-numpy.random) with a fixed seed for repeatable, consistent execution.

For the power one, the parameter is now a pennylane numpy array instead of a list of lists.  NumPy arrays are the preferred method for efficiency, and that should be reflected in our benchmarks.

### CODE REVIEW

File: benchmarks/benchmark_functions/pl-braket-pipeline.py [hunk 1]
```diff
@@ -69,8 +69,10 @@ def circuit(params_):
         qml.templates.BasicEntanglerLayers(params_, wires=range(n_wires))
         return qml.expval(qml.PauliZ(0))
 
-    params = random((n_layers, n_wires))
-    params = pnp.array(params, requires_grad=True)
+
```
Comment (antalszava) [inline benchmarks/benchmark_functions/pl-braket-pipeline.py]:
```suggestion
```

File: benchmarks/benchmark_functions/pl-braket-pipeline.py [hunk 2]
```diff
@@ -125,7 +127,8 @@ def benchmark_power(dev_name, s3=None):
     n_layers = 1
     graph = nx.complete_graph(n_wires)
 
-    params = [[0.5] * n_layers, [0.5] * n_layers]
+
+    x = 0.5 * pnp.ones((n_layers, 2))
```
Comment (antalszava) [inline benchmarks/benchmark_functions/pl-braket-pipeline.py]:
```suggestion
    params = 0.5 * pnp.ones((n_layers, 2))
```

**Review by antalszava — APPROVED** [review]
:confetti_ball: The second one needs an update as per the suggestion, but looks good otherwise!

File: benchmarks/benchmark_functions/pl-braket-pipeline.py (hunk 2 above)
Comment (albi3ro) [reply benchmarks/benchmark_functions/pl-braket-pipeline.py]: oops
